index.js

```diff
@@ -376,5 +377,48 @@ module.exports = {
   
   _isModuleUnification() {
     return (typeof this.project.isModuleUnification === 'function') && this.project.isModuleUnification();
+  },
+
+  _updateAppFilesInManifest(appDir) {
+    const pkgPath = path.join(appDir, 'package.json');
+    const pkg = require(pkgPath);
+    const {
+      fastboot
+    } = pkg;
+
+    if(fastboot) {
+      const { manifest: { appFiles: manifestAppFiles } } = fastboot;
+      const appFilePath = path.resolve(appDir, manifestAppFiles[0]);
```

Comment:
Does this error if `manifestAppFiles` is an empty array? Would it be empty in the embroider case?

Reply:
Thank you @rwjblue.  That is correct. if `manifestAppFiles` is empty, the `resolve` method will throw an error. But In the first stage of the build of embroider we will always expect generated app paths to be present.

But,  I added a defensive check that if `manifestAppFiles` is an Array and the first element is a string to make sure there is no error thrown here.